projects/power_select/rules.mk

```diff
@@ -6,6 +6,7 @@
 # $(T)_SRC: $(T)_DIR/src{/$(PLATFORM)}/*.{c,s}
 
 # Specify the libraries you want to include
-$(T)_DEPS := ms-helper ms-common codegen-tooling
+$(T)_DEPS := ms-common
 
-$(T)_test_power_selection_MOCKS := adc_read_raw
\ No newline at end of file
+$(T)_test_power_select_MOCKS := adc_read_converted_pin gpio_get_state gpio_set_state
```

Comment:
Sorry do you mock these?

Reply:
Yeah, lines 63/93/115 in `test_power_select.c`